libraries/AP_HAL_ChibiOS/hwdef/EAH743QF/defaults.parm

```diff
@@ -0,0 +1,7 @@
+BATT_MONITOR, 4
+BATT_VOLT_PIN, 10
+BATT_CURR_PIN, 11
+BATT_VOLT_MULT, 11.1
+BATT_AMP_PERVLT, 100
+
+SERIAL4_PROTOCOL, 2 # Telemetry on SiK Radio
```

Comment:
conflict, Serial $ is RC input
should add OSD_TYPE2 5 as only entry to enable HD VTX OSD